src/AcceptanceTests/NServiceBus.ClaimCheck.AzureBlobStorage.AcceptanceTests.csproj

```diff
@@ -9,14 +9,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.21.2" />
-    <PackageReference Include="GitHubActionsTestLogger" Version="2.4.1">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="GitHubActionsTestLogger" Version="2.4.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
     <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="9.2.2" GeneratePathProperty="true" />
```

Comment:
I left it on 9.2.2, because NServiceBus.ClaimCheck uses 9.2.2

Reply:
I changed this to 9.2.3 in the tweaks I pushed up. We should always be using the latest version.

I also added an NServiceBus reference to the main project here because while there is a transitive reference coming in from NServiceBus.ClaimCheck, all downstream packages should have a direct NServiceBus reference as well.